interfaces/HO-Portal/src/assets/i18n/en.json

```diff
@@ -597,9 +597,11 @@
       "pageTitle": "User",
       "change-password": {
         "title": "Change password",
+        "existing-password": "Existing password",
```

Comment:
I would call this 'Current password' unless already otherwise agreed with someone else from our team.